### ISSUE TITLE
Remove FeatureGroup after drawing (DrawControl)

### ISSUE THREAD

**Issue by fitoprincipe** [jupyter-widgets/ipyleaflet#169]
I need to add the drawn shape to the map as a layer and remove the created FeatureGroup after every drawing, without having to do it manually. The callback function would be as simple as:

```
def handle_draw(self, action, geo_json):
    new_poly = GeoJSON(data=geo_json)
    m.add_layer(new_poly)
```
But doing only this the FeatueGroup persists.
The behavior would be the same as in http://leaflet.github.io/Leaflet.draw/docs/examples/full.html

**Comment by fitoprincipe** [comment]
I see now this is a duplicate of https://github.com/jupyter-widgets/ipyleaflet/issues/68#issuecomment-391160071

**Comment by martinRenou** [comment]
Closed by #194
